hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -450,6 +436,9 @@ private void sanityCheckConfigs() {
     Preconditions.checkArgument((singleFactor + multiFactor + memoryFactor) == 1,
       SINGLE_FACTOR_CONFIG_NAME + ", " + MULTI_FACTOR_CONFIG_NAME + ", and "
         + MEMORY_FACTOR_CONFIG_NAME + " segments must add up to 1.0");
+    if (this.persistenceChunkSize <= 0) {
```

Comment:
Is this change related?

Reply:
This check was previously being done outside of sanityCheckConfigs (see line 358 on the original version), since it is a sanity check, though it makes sense to move it here.